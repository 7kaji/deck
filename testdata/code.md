# Inline Code

---

# Title `with code`

## Hello `code`

- Hello `code`
- `code` world

`code`

