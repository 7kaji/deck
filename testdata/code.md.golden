[
  {
    "layout": "",
    "titles": [
      "Inline Code"
    ]
  },
  {
    "layout": "",
    "titles": [
      "Title `with code`"
    ],
    "subtitles": [
      "Hello `code`"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "Hello "
              },
              {
                "value": "code",
                "code": true
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "code",
                "code": true
              },
              {
                "value": " world"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "code",
                "code": true
              }
            ]
          }
        ]
      }
    ]
  }
]